src/main/java/org/terasology/logic/inventory/InventoryUtils.java

```diff
@@ -135,6 +135,10 @@ private static boolean hasSameAttributes(EntityRef from, EntityRef to) {
         return true;
     }
 
+    public static boolean isStackable(ItemComponent item) {
```

Comment:
I believe a check for stackability is performed in other places as well. Whilst on this, consider switching them to use this method instead.

Reply:
I don't want to do this in this PR for all other occurrences. I think this should be done in a different PR (probably moving this method to that PR, too).